apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/SpannerEFSnippets.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using Google.Cloud.ClientTesting;
+using Google.Cloud.Spanner.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Google.Cloud.EntityFrameworkCore.Spanner.Snippets
+{
+    [SnippetOutputCollector]
+    public class SpannerEfSnippets
+    {
+        private static IConfiguration Config { get; set; }
+
+        const string DefaultConnectionString =
+            "Data Source=projects/cloud-sharp-jenkins/instances/spannerinstance/databases/chips";
```

Comment:
Should this project be configurable? I don't think we should hard code projects in.

Reply:
That's just a fallback in case the config entry doesnt exist -- It pulls from env variables.

Reply:
We don't have any coded projects anywhere else in the repo.  I don't think it's good to start using them.  If anyone outside our team uses this it will fail.  All the other bits just throw it TEST_PROJECT isn't set.

Reply:
Fixed.